ansible/roles/lvm-init/tasks/main.yml

```diff
@@ -80,7 +82,7 @@
         - /var/lib/cloud/scripts/per-boot
 
     - name: Data partition             | Auto resize LVM
-      when: enable_lvm == "true"
+      when: enable_lvm == "true" and not single_disk
```

Comment:
It seems it'll be better to use` block` here:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html

```
block:
****
when: not single_disk
```

Reply:
Done!